content/influxdb/cloud-dedicated/query-data/troubleshoot.md

```diff
@@ -0,0 +1,194 @@
+---
+title: Troubleshoot responses and errors
+description: >
+  Learn how to handle responses and troubleshoot errors encountered when querying {{% cloud-name %}} with Flight+gRPC and Arrow Flight clients.
+weight: 201
+menu:
+  influxdb_cloud_dedicated:
+    name: Troubleshoot responses and errors
+    parent: Query data
+influxdb/cloud-dedicated/tags: [query, sql, influxql]
+alt_engine:
+---
+
+Learn how to handle responses and troubleshoot errors encountered when querying {{% cloud-name %}} with Flight+gRPC and Arrow Flight clients.
+
+<!-- TOC -->
+
+- [InfluxDB Flight responses](#influxdb-flight-responses)
+  - [Schema](#schema)
+    - [Example](#example)
+  - [InfluxDB error codes](#influxdb-error-codes)
+  - [Troubleshoot errors](#troubleshoot-errors)
+    - [Internal Error: Received RST_STREAM](#internal-error-received-rst_stream)
+    - [Internal Error: stream terminated by RST_STREAM with NO_ERROR](#internal-error-stream-terminated-by-rst_stream-with-no_error)
+    - [Invalid Argument Error: bucket <BUCKET_ID> not found](#invalid-argument-error-bucket-bucket_id-not-found)
+      - [Unauthenticated: Unauthenticated](#unauthenticated-unauthenticated)
+    - [Unauthenticated: read:<BUCKET_ID> is unauthorized](#unauthenticated-readbucket_id-is-unauthorized)
+
+## InfluxDB Flight responses
+
+{{% cloud-name %}} provides an InfluxDB-specific Arrow Flight service that uses gRPC Remote Procedure Calls (gRPC) to transport query result data in Arrow format.
+Flight defines a set of [RPC methods](https://arrow.apache.org/docs/format/Flight.html#rpc-methods-and-request-patterns) for servers and clients.
+Using gRPC, a Flight client (for example, `influx3` or an InfluxDB v3 client library) calls an InfluxDB Flight RPC server method to send a request to the server.
+In gRPC, every call returns a status object that contains an integer code and a string message.
+While gRPC defines the integer [status codes](https://grpc.github.io/grpc/core/status_8h.html) and definitions for servers and clients,
+a server can choose which status to return for an RPC call.
+During a request, the gRPC client and server may each return a status independent of the other.
```

Comment:
I don't understand what "a status independent of the other" means. 

Maybe it is trying to say something like errors can come from the server (e.g. a processing failure) or from the client itself (e.g. a network failure)?

Reply:
Yes, I should be clearer here.